src/scripts/main.js

```diff
@@ -2,6 +2,24 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const body = document.querySelector('body');
+  const block = document.createElement('div');
+  const hTwo = document.createElement('h2');
```

Comment:
```suggestion
  const heading = document.createElement('h2');
```
Краще використовувати більш зрозумілі назви